server/api/v2/sendMessage/sendMessagePost.ts

```diff
@@ -96,8 +96,8 @@ export const postMessage = async (req: RequestWithUser, res: Response): Promise<
 		embed.setImage(req.body.attachment);
 	}
 
-	req.body.message && embed.setDescription(`Content:\n${req.body.message}`);
-	embed.setAuthor(req.body.name ? req.body.name : "Anonymous", AVATAR, "https://pxseu.com/msg");
+	req.body.message && embed.setDescription(req.body.message);
+	embed.setAuthor(req.body.name || "Anonymous", AVATAR, "https://pxseu.com/msg");
```

Comment:
we tu daj `??` bo chyba lepsze bedzie

Reply:
jak ja nie wiem co to

Reply:
Nullish coalescing operator

Reply:
mulisz co

Reply:
albo nie
nwm

Reply:
chyba git

Reply:
nie

Reply:
bo jak ktoś da nick jako `""` to bot da to jako tytuł

Reply:
i tak robile null checki wiec no

Reply:
https://github.com/pxseu/pxseu.com/blob/2848bacc394e5934dbf1aa63a192d3c5bb3caa9a/server/api/v2/sendMessage/sendMessagePost.ts#L66 
tu masz